rootfs/scheduler/__init__.py

```diff
@@ -667,6 +679,44 @@ def _delete_rc(self, namespace, name):
             error(resp, 'delete ReplicationController "{}" in Namespace "{}"',
                   name, namespace)
 
+    def _healthcheck(self, controller, path='/', port=80, delay=30, timeout=1):
+        # FIXME this logic ideally should live higher up
+        if controller['spec']['selector']['type'] not in ['web', 'cmd']:
```

Comment:
This isn't the only place it happens (I did a similar thing around making apps routable) so I didn't want to push too hard just yet but I'll have a look, I believe `_deploy_app` isn't the only place that this this up